pulseapi/entries/models.py

```diff
@@ -18,6 +18,25 @@ def entry_thumbnail_path(instance, filename):
 
 
 # Create your models here.
+class ModerationState(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return str(self.name)
+
+
+def get_default_moderation_state():
+    """
+    Helper function to ensure there is a default
+    ModerationState that can be tacked onto Entries.
+    """
+
+    default_state = ModerationState.objects.all()[0].id
```

Comment:
Scratch that, I didn't read far enough in another file to understand the migrations.